package/Dockerfile

```diff
@@ -13,7 +13,7 @@ WORKDIR /var/lib/rancher
 
 ARG ARCH=amd64
 ARG IMAGE_REPO=rancher
-ARG SYSTEM_CHART_DEFAULT_BRANCH=release-v2.5
+ARG SYSTEM_CHART_DEFAULT_BRANCH=dev-v2.5.9
```

Comment:
Theres a few more, see release checklist
cc @PennyScissors

Reply:
@cbron The release checklist mentions 2 places, and I just updated the PR to change both of them.

Reply:
Yeah, the checklist mentions the 2 places @jiaqiluo updated. Is there another place that maybe we missed in the docs?

Reply:
Looks like it's those 2 places for system-charts: https://github.com/rancher/rancher/commit/a159bd1cc1e8037d0bfc25b3a1ed4d0e6d6a804f#diff-c55d1ee74f32cd4b9eeeceaec981a180f52e761acb158a5b0b0e088c7725c9ca